### PR TITLE
Fitness api

### PR DESCRIPTION
**Pull request by inanimate631**

### CODE REVIEW

File: backend/authController.js [hunk 1]
```diff
@@ -88,5 +89,84 @@ class authController {
...
+  async createExerciseList(req, res) {
+    const options = workoutHelpers.workoutHelpers.options;
+    const arrayMuscule = [];
+    const nameArr = [];
+    const muscule = await fetch(
+      `${workoutHelpers.workoutHelpers.exerciseApi}muscles/`,
+      options
+    );
+    const dataMuscule = await muscule.json();
+    for (let i = 0; i < 5 /*switch to dataMusculeNum*/; i++) {
```
Comment (RetiredHedgehog) [inline backend/authController.js:101]: Seems like you forgot to switch to dataMusculeNum.

File: backend/authRouter.js [hunk 2]
```diff
@@ -21,5 +21,8 @@ router.post(
 router.post('/login', controller.login);
 router.get('/users', roleMiddleware(['ADMIN']), controller.getUsers);
 router.put('/updateUser', controller.changeValue);
+router.get('/exercisesList', controller.createExerciseList);
+router.get('/exercises/:name', controller.createExercise);
+router.get('/worckout/:name', controller.createWorckout);
```
Comment (RetiredHedgehog) [inline backend/authRouter.js:26]: _workout_, without _c_

File: backend/workout.helpers.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+const apiKey = [
+  'bdeff97547msh9e2fb2cf4487c88p1c2cdejsn329751128532',
+  '29e98089a8msh3fd3ed66dc3490ap16249fjsn0f190f5481b5',
+  '381a351d1fmsh49002bfcdd619a8p1fb7b6jsn2dfc9e093608',
+];
+const MAX_API_KEY = 3;
+function getRandomInt(max) {
+  return Math.floor(Math.random() * max);
+}
```
Comment (RetiredHedgehog) [inline backend/workout.helpers.js:9]: `Math.random()` returns a number in range `[0, 1)`. `1` is excluded, therefore the number returned by `getRandomInt(max)` will always be less then `max`.

Use this instead:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random#getting_a_random_integer_between_two_values_inclusive

File: backend/authController.js (hunk 1 above)
Comment (inanimate631) [reply backend/authController.js:101]: If you switch to length, then 23 requests will go at a time, so it's better to let it be 5

File: backend/workout.helpers.js (hunk 3 above)
Comment (inanimate631) [reply backend/workout.helpers.js:9]: but I need to get the number up to 3

Comment (RetiredHedgehog) [reply backend/workout.helpers.js:9]: Ah, my bad, I forgot, that array's indexes start at 0

**Review by RetiredHedgehog — APPROVED** [review]

**Review by Doroshkeviches — APPROVED** [review]

**Pull request merged**